host/containerinit/init.go

```diff
@@ -513,6 +584,19 @@ func containerInitApp(c *Config) error {
 	init.changeState(StateRunning, "", -1)
 
 	init.mtx.Unlock() // Allow calls
+	// monitor services
+	for _, port := range c.Ports {
+		if port.Service == nil {
+			continue
+		}
+		hb, err := monitor(port, c.Env)
+		if err != nil {
+			// TODO: handle err
```

Comment:
I am unsure how to handle this. Should the entire application fail, if the Register fails to spin up for some reason, or should we simply skip it (like we do now) and let the application run?